pkg/cri/server/service.go

```diff
@@ -110,10 +111,12 @@ type criService struct {
 	// allCaps is the list of the capabilities.
 	// When nil, parsed from CapEff of /proc/self/status.
 	allCaps []string // nolint
+
+	metrics collector
```

Comment:
I'm on the fence on whether or not this should just be a package level var.

Reply:
Ah but which package...

We might wish to consider the portion of the network setup that occurs inside a runtime engine for vm guests.. and if so we might want to have different per runtime type metrics. We may also wish to have network setups for the core sandbox service?  

Maybe leave it the way it is with a TODO for refactoring networking (and metrics) into it's own package with a set of interfaces and it's own config. I'm good either way.